.github/workflows/Tests.yml

```
@@ -323,35 +455,118 @@ jobs:
       if: ${{ runner.os }} != "Linux"
       run: |
         if [ $OS == macos-latest ] ; then ./cc-test-reporter after-build --exit-code 0 || true ; else echo "SKIP code climate" ; fi ;
```

```suggestion
        if [ $OS == macos-latest ] ; then ./cc-test-reporter after-build --exit-code 0 || true ; else echo "::warning file=.github/workflows/Tests.yml,line=457,endLine=457,title=SKIPPED::SKIP Code-Climate" ; fi ;
```

This would be improved by using a message format GHA will annotate.